met-web/src/components/comments/admin/review/CommentReview.tsx

```diff
@@ -72,10 +79,20 @@ const CommentReview = () => {
         }
     };
 
+    const extract_staff_note = async () => {
+        staffNote.length === 0
```

Comment:
I'd use this the other way around setUpdatedStaffNote(staffNote.len... ? staffNote : default..)

Reply:
addressed it, thanks.